README.md

```diff
@@ -12,24 +12,26 @@ https://pattern-library.dequelabs.com is a resource for an accessible UI library
 
 https://vuetifyjs.com/ is an extremely well-built Vue UI library to use as a reference for building components.
 
-This project follows the Vue style guide: https://vuejs.org/v2/style-guide/ and the https://standardjs.com/ style guide.
```

Comment:
This line should stay as it refers to code style, not visual style.

Reply:
^^^

Reply:
fixed: moved to styleguide section just to keep it all together

Reply:
I don't think it belongs in the styleguide section because they're two different things. The Vue style guide is about how to write Vue code. The generated style guide is really more the documentation of our UI framework. Maybe we should just call it documentation instead to avoid confusion.